geonode/templates/user_messages/inbox.html

```diff
@@ -40,17 +40,23 @@ <h2 class="page-title">{%  trans "Messages" %}</h2>
 
 {% block extra_script %}
 <script type="text/javascript">
-$('.message_delete_btn').click(function(event) {
-  form = $(this).parent()[0];
-  $.ajax({
-    type: "POST",
-    url: $(form).attr('action'),
-    data: $(form).serialize(),
-    success: function() {
-      $('#inbox').load(window.location.pathname + ' #inbox');
-      $('#all').load(window.location.pathname + ' #all');
-    }
-  });
+$('.tab-content').click(function(event) {
```

Comment:
Attaching the listener to the containing element - the bug before occurred because we reloaded the elements that had the listeners attached to them.